code/resources/css/style.css

```diff
@@ -0,0 +1,511 @@
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/*
+COLOR PALETTE: 
+
+Main colors: 
+
+Dark purple: #240747;
+Red: #eb2632; 
+
+Dust Palette: 
+
+Wine: #792A17;
+Grey: #7D7484;
+Light purple: #B1A8B9;
+Darkest purple: #FDE9FF;
+
+*/
+
+/* ----------------------------------------------- */
+/*                GLOBAL COMPONENTS                */
+/* ----------------------------------------------- */
+
+body {
+  background-color: #fff;
+  font-family: 'Flamenco', serif;
+  color: #fff;
+}
+
+/* ----------------------------------------------- */
+/*                    HEADER                       */
+/* ----------------------------------------------- */
+
+.hero {
+  position: relative;
+  width: 100%;
+  height: 200px;
+  text-align: center;
+  overflow: hidden;
+  z-index: 300;
+}
+
+.hero .hero__overlay {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  z-index: 2;
+  background-color: #240747;
+  opacity: 0.5;
+}
+
+.hero__video {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
+  object-position: center;
+  object-fit: cover;
+}
+
+.hero__nav-container__logo {
+  height: 10vw;
+  width: auto;
+  float: left;
+  z-index: 10;
+}
+
+.hero__nav-container__nav-bar__nav-link {
+  display: none;
+  padding: 15px 20px;
+  font-size: 20px;
+  text-decoration: none;
+  }
+
+.hero__contents {
+  position: relative;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
You can use transform instead of margin to make sure your image/video is responsive regardless of whether you've added breakpoints or not. Here's a bit of info about that property: https://www.w3schools.com/cssref/css3_pr_transform.asp And people here also gave some useful answers about it: https://stackoverflow.com/questions/46184458/transform-translate-50-50/46184660